src/pages/console/formbuilder/overview.mdx

```diff
@@ -82,10 +105,42 @@ This JSON will render the following form:
 
 ![Form rendered from JSON showing headings and fields with types inferred from key pair values](/images/console/formbuilder/overview6.png)
 
-From the JSON object, Amplify Studio can automatically
-- set the correct types for **Email address** and **Number**
-- set **Animals** to receive multiple values
-- set **Active** and **Enabled** to switches
+From the JSON object, Amplify Studio can automatically...
+- ...set the correct types for **Email address** and **Number**
+- ...set **Animals** to receive multiple values
+- ...set **Active** and **Enabled** to switches
+
+When you are satisfied with this form, you can convert it to a connected form by [binding to a data model](#form-binding), or you [can render it in your app](#render-react-form-in-your-app).
+
+</Block>
+
+<Block name="New blank form">
+
+Amplify Studio allows you to build unconnected forms completely from scratch with a visual interface. To generate a basic, brand-new form:
+
+1. In the [Amplify Studio Console](/console/adminui/start/#logging-in-and-creating-an-app), navigate to **UI Library**
+1. In the left-hand nav bar, next to the **Forms** header, select **New**
+![Red box around the "new" section of Form Builder](/images/console/formbuilder/overview7.png)
+1. Give your form a name, and select [Create or Update](#create-and-update-forms)
+1. Leave **Black form** selected, and click **Create form**
+
+![New form page, showing the options available when creating a new form](/images/console/formbuilder/overview2.png)
+
+After clicking **Create form**, you will be directed to the Edit Form page. Select the **plus sign**, and select an input element to add to your form. 
+
+![Image of the Form Creation page with the blue bar and plus sign indicating the ability to add input elements to the form](/images/console/formbuilder/overview3.png)
+
+You can continue to customize your form inputs, and when you are satisfied, you can convert it to a connected form by [binding to a data model](#form-binding), or you [can render it in your app](#render-react-form-in-your-app).
+
+</Block>
+
+</BlockSwitcher>
+
+<Callout info>
+
+Alternatively, to get started without logging into AWS, [you can build unconnected forms in the Amplify Sandbox](https://sandbox.amplifyapp.com/ui-library). 
+
+</Callout>
 
 ## Render React form in your app
```

Comment:
do you think it may be better to move this section to a different page?

Reply:
Talked about this, we might revisit in the future if we receive feedback that this is awkward or difficult to understand.